tutorindigo/templates/indigo/lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -59,9 +59,8 @@
           </svg><span class="sr">${_("Post a Facebook message to say you've enrolled in this course")}</span>
       </a>
       <a href="${email_link}" class="share">
-        <svg xmlns="http://www.w3.org/2000/svg" width="22" height="18" viewBox="0 0 22 18" fill="none">
-          <path d="M3 1H19C20.1 1 21 1.9 21 3V15C21 16.1 20.1 17 19 17H3C1.9 17 1 16.1 1 15V3C1 1.9 1.9 1 3 1Z" fill="#9CA3AF" stroke="#F2F7F8" stroke-linecap="round" stroke-linejoin="round"/>
-          <path d="M21 3L11 10L1 3" stroke="#F2F7F8" stroke-linecap="round" stroke-linejoin="round"/>
+        <svg xmlns="http://www.w3.org/2000/svg" width="22" height="17" viewBox="0 0 22 17" fill="none">
```

Comment:
We should definitely highlight this change with a comment. Even better: make a PR upstream to avoid hard-coding the SVG right in the HTML file.

Reply:
Regis, we are considering using the new SVG icons to enhance the UI. If rendering takes more page load time or is heavy, we can remove or update these. Here you can see the black icons used in edx-platform (they have been used as span tags with `.fa .icon` classes). As you can see edx-platform is using a bird icon for Twitter but we've replaced it with `X` icon. 

<img width="360" alt="Screenshot 2024-07-19 at 6 06 35 PM" src="https://github.com/user-attachments/assets/8b02f51a-8b51-4de1-8840-323d7b0c9c40">

<img width="310" alt="Screenshot 2024-07-19 at 6 02 02 PM" src="https://github.com/user-attachments/assets/178e99b0-c675-472f-a6e2-c8099bd173c2">